systems/plants/test/debugManipulatorDynamics.cpp

```diff
@@ -0,0 +1,58 @@
+#include "RigidBodyManipulator.h"
+#include "URDFRigidBodyManipulator.h"
+#include "drakeGeometryUtil.h"
+#include <iostream>
+#include <cstdlib>
+#include <random>
+#include <memory>
+
+using namespace std;
+using namespace Eigen;
+int main()
+{
+  URDFRigidBodyManipulator* model = loadURDFfromFile("examples/Atlas/urdf/atlas_minimal_contact.urdf");
+  if(!model)
+  {
+    cerr<<"ERROR: Failed to load model"<<endl;
+  }
+  int gradient_order = 1;
+  int nq = model->num_dof;
+  int nv = model->num_velocities;
+
+  default_random_engine generator;
+  uniform_real_distribution<double> uniform;
+
+  for (vector<unique_ptr<RigidBody> >::const_iterator it = model->bodies.begin(); it != model->bodies.end(); ++it) {
+    RigidBody& body = **it;
+    double mass = uniform(generator);
+    Matrix3d moment_of_inertia = Matrix3d::Random();
+    moment_of_inertia *= moment_of_inertia;
+    Vector3d com = Vector3d::Random();
+
```

Comment:
I just needed the mass/inertia to be initialized, yeah. Having URDFRigidBodyManipulator initialize those would be great.

It's not really a test since it's not checking anything, which is why I prefixed it with debug. It just made debugging much easier since it didn't require me to use Matlab to call it, allowing me to use the debugger.